py-scripts/lf_mixed_traffic.py

```diff
@@ -2065,6 +2095,26 @@ def generate_all_report(self):
                 self.lf_report_mt.set_csv_filename(ftp_graph2)
                 self.lf_report_mt.move_csv_file()
                 self.lf_report_mt.build_graph()
+                if self.dowebgui and self.get_live_view:
```

Comment:
Could probably move this to a dedicated test house-specific report generation function, but up to you.